modules/sdk-core/src/bitgo/enterprise/enterprise.ts

```diff
@@ -148,4 +148,23 @@ export class Enterprise implements IEnterprise {
       challenge
     );
   }
+
+  /**
+   * Fetches the existing TSS ECDSA enterprise challenge if one exists.
+   * Can be used with uploadAndEnableTssEcdsaSigning to re-sign the
+   * enterprise challenge with new signatures.
+   */
+  async getExistingTssEcdsaChallenge(): Promise<NTilde> {
```

Comment:
```suggestion
  async getTssEcdsaChallenge(): Promise<NTilde> {
```

`existing` seems redundant here.

Reply:
It's just to clarify it doesn't create a new one if it doesnt exist